### PR TITLE
Properly detect SVG clickables with no text

### PR DESCRIPTION
**Pull request by obto**
This is the PR we discussed last week.

Fixes the issue where SVG clickables of any kind were being detected

### CODE REVIEW

**Comment by obto** [comment]
@rviscomi Is it possible to merge this before the next run?

**Comment by rviscomi** [comment]
Sorry this slipped past me. The October crawl started a couple days ago, but it wouldn't be completed until the end of the month anyway. That wouldn't leave us much time to get the results analyzed and updated in the chapter before launch. :/

**Comment by obto** [comment]
Hmm since the detection of links using font icons was accurate I think I'll do the following to get a rough estimate for the Almanac then:

1. Run an analysis to find out what % of pages (which have buttons and links) have any svgs on them at all
2. Remove that % from the metric result we already have... which should effectively give me the % of pages using clickables with only a font icon

This should give me a pretty good lower bound, and I can go back and edit the Almanac at a later point when we re-run it in the future.

**Comment by rviscomi** [comment]
SGTM 👍

**Comment by obto** [comment]
Great :+1: 

Is this PR alright to merge then?

**Comment by rviscomi** [comment]
It is, but there's another step which is for me to sync it with the HTTP Archive test server. I'll hold off on merging until I have time to sync before the next crawl as a reminder.

**Comment by rviscomi** [comment]
Wow I had forgotten this change existed! I'll merge now to get it in before the August crawl. Sorry @obto! 😅

File: custom_metrics/almanac.js [hunk 1]
```diff
@@ -278,19 +278,42 @@ return JSON.stringify({
     return parsedNodes;
   })(),
   '12.11': (() => {
+    function containsAnSvg(element) {
+      var children = Array.from(element.childNodes);
+      return !!children.find((child) => {
```
Comment (rviscomi) [inline custom_metrics/almanac.js:283]: Actually could this be simplified to

```js
function containsAnSvg(element) {
  return !!element.querySelector('svg');
}
```

**Comment by obto** [comment]
@rviscomi No worries. I've included this change in the branch I'm working on right now for migrating as many responsebody queries into custom metrics. So we can close this PR request.

File: custom_metrics/almanac.js (hunk 1 above)
Comment (obto) [reply custom_metrics/almanac.js:283]: Good call. I'll update it in my other branch

**Pull request closed without merging**